readme.md

```diff
@@ -167,202 +178,93 @@ JSPath.apply('.books..name', doc);
 ```
 
 ###Predicates###
-JSPath predicates allow you to write very specific rules about items you'd like to select when constructing your expressions.
-Predicates are filters that restrict the items selected by location path. There're two possible types of predicates: object and positional.
+JSPath predicates allow you to write very specific rules about items you'd like to select when constructing your path expression. Predicates are filters that restrict the items selected by the location path. There're two possible types of predicates: [object](#object-predicates) and [positional predicates](#positional-predicates).
 
 ###Object predicates###
-Object predicates can be used in a path expression to filter a subset of items according to boolean expressions working on a properties of each item.
-Object predicates are embedded in braces.
+Object predicates can be used in a path expression to filter a subset of items according to boolean expressions working on a properties of each item. All object predicates are parenthesized by curly brackets (`{` and `}`).
 
-Basic expressions in object predicates:
-  * numeric literals (e.g. 1.23)
-  * string literals (e.g. "John Gold")
-  * boolean literals (true/false)
-  * subpathes (e.g. .nestedProp.deeplyNestedProp)
+In JSPath these basic expressions can be used inside an object predicate:
+  * numeric literals (e.g. `1.23`)
+  * string literals (e.g. `"John Gold"`)
+  * boolean literals (`true` and `false`)
+  * subpathes (e.g. `.nestedProp.deeplyNestedProp`)
 
-JSPath allows to use in predicate expressions following types of operators:
-  * comparison operators
-  * string comparison operators
-  * logical operators
-  * arithmetic operators
+Furthermore are following types of operators valide inside an object predicate:
```

Comment:
Seems should be "Furthermore the following types of operators are valid inside an object predicate:"